AnnotatoBackend/Sources/App/Persistence/SharedLibraryModels+Persistable/Annotation+Persistable.swift

```diff
@@ -7,10 +7,19 @@ extension Annotation: Persistable {
         let handwritingParts = managedEntity.annotationHandwritingEntities.map(AnnotationHandwriting.fromManagedEntity)
         let parts: [AnnotationPart] = textParts + handwritingParts
 
+        let selectionBox: SelectionBox
+        if let selectionBoxEntity = managedEntity.selectionBox {
+            selectionBox = SelectionBox.fromManagedEntity(selectionBoxEntity)
+        } else {
+            // Will never come to this part. But it has to be optional because we use @OptionalChild
```

Comment:
why must we use optional child if it's not optional? is this a fluent thing?

Reply:
Yea there's no compulsory child in fluent
And another alternative is Children which gives an array instead of an optional